yolov5/yolov5.cpp

```diff
@@ -8,17 +8,25 @@
 #include "calibrator.h"
 #include "preprocess.h"
 
-#define USE_FP16  // set USE_INT8 or USE_FP16 or USE_FP32
+#define USE_FP32  // set USE_INT8 or USE_FP16 or USE_FP32
+#define USE_CLASSIFY // set USE_CLASSIFY or USE_DETECT to serialize with classification or detection network
```

Comment:
@xiang-wuu  I know that,  but please, I still think it's better to have separate .cpp file. And also separate .py file.

Because for users,  it will be easier for user to understand the code structure.